recipe/activate-gcc.sh

```diff
@@ -185,4 +187,21 @@ else
     diff -U 0 -rN /tmp/old-env-$$.txt /tmp/new-env-$$.txt | tail -n +4 | grep "^-.*\|^+.*" | grep -v "CONDA_BACKUP_" | sort
     rm -f /tmp/old-env-$$.txt /tmp/new-env-$$.txt || true
   fi
+
+  # fix prompt for zsh
+  if [[ -n "${ZSH_NAME}" ]]; then
```

Comment:
```suggestion
  if [[ -n "${ZSH_NAME:-}" ]]; then
```

Reply:
Can you make the same change in clang recipe? Need this for shell sessions with `-u` set.

Reply:
Yep. Will fix!

Reply:
done!